distribution/std-lib/Standard/src/Image/Codecs.enso

```diff
@@ -1,57 +1,64 @@
 from Standard.Base import all
+
 import Standard.Base.System.File
-import Standard.Image.Data.Image
 import Standard.Image.Codecs.Internal
+import Standard.Image.Data.Image
 
 polyglot java import org.enso.image.Codecs as Java_Codecs
 
 type Read_Flag
 
-    ## Read image with alpha channel, otherwise it gets cropped.
+    ## Read the image with its alpha channel, otherwise the channel gets cropped.
     type Read_Alpha_Channel
 
-    ## Always convert image to the single channel grayscale image.
+    ## Always convert the image to a single channel grayscale image.
     type Read_Grayscale
 
-    ## Use the gdal driver for loading the image.
+    ## Use Geographic Data Abstraction Library (GDAL) driver to load images in
+       geospatial raster data formats.
     type Read_Gdal
 
 type Write_Flag
 
-    ## For JPEG, it can be a quality from 0 to 100 (the higher is the better).
-       Default value is 95.
-    type Write_Jpeg_Quality value
+    ## For a JPEG, this can be a quality from 0 to 100 (the higher, the better).
+       The default value is 95.
+    type Write_Jpeg_Quality value=95
 
-    ## Enable JPEG features. Disabled by default.
+    ## Enable progressive JPEG compression format. Disabled by default.
     type Write_Jpeg_Progressive
 
-    ## Enable JPEG features. Disabled by default.
+    ## Enable optimized JPEG encoding algorithms. Disabled by default.
     type Write_Jpeg_Optimize
 
     ## Separate luma quality level, 0 - 100, default is 0 - don't use.
-    type Write_Jpeg_Luma_Quality value
+    type Write_Jpeg_Luma_Quality value=0
 
     ## Separate chroma quality level, 0 - 100, default is 0 - don't use.
-    type Write_Jpeg_Chroma_Quality value
+    type Write_Jpeg_Chroma_Quality value=0
 
     ## For PNG, it can be the compression level from 0 to 9. A higher value
        means a smaller size and longer compression time. Default value is 3.
```

Comment:
You don't need to document the defaults here? As far as I know the defaults get extracted and documented by the suggestions mechanism. Is that not correct?

Reply:
You're right, the default value shows in suggestions. I can remove it from the docstring.